packages/common/styles/_styles.scss

```diff
@@ -0,0 +1,13 @@
+@use './tokens';
+
+.dsco-text {
+  color: tokens.$dsco-font-color;
+  font-size: tokens.$dsco-font-size;
+  line-height: 20px;
```

Comment:
line-height is generally a unitless value, if we've got a default font size of 16px, this should be `1.25`

Reply:
ooo, thank you! i grabbed this from our branding guidelines so i'll talk to mark about this too